src/app-pages/map/components/map-navbar/LocationTree.js

```diff
@@ -46,7 +46,7 @@ const LocationTree = ({
         locationId: e.id,
         lat: e.latitude,
         lon: e.longitude,
-        zoom: e.zoom_depth ? Math.round( e.zoom_depth * 1.5 ) : defaultMapParams.zoom,
+        zoom: e.zoom_depth ? Math.round( e.zoom_depth * 1.5 ) : defaultMapParams.locationZoom,
```

Comment:
We should remove locationZoom and use the defaultMapPrams.zoom instead. If a location is in the location tree, but not on the map, the map will zoom into a random location. This happens if you select North Atlantic -> Baltimore -> Allegheny River (or any river under Baltimore).